### DISCUSSION TITLE
Performers Name vs. Username

### DISCUSSION THREAD

**Discussion by GreyBeardy** [DIGITALCRIMINALS/OnlyFans#370] [Q&A]
Hi,

I am aware of the {model_username} = The model's username. for paths and filenmaes.
For Isabella Miller XXX this gives the username isabellemillerx - I was wondering if there is any way to use the "Isabella Miller XXX" as an option for folders and filenames ?

Right now only @isabellermillerx is used - unless I missed something.

thanks.

**Comment by DIGITALCRIMINAL** [comment]
I could add in a "name" option, but you're still going to have to use a unique identifier for the script, so you could use the account's id along with the name.

**Comment by GreyBeardy** [comment]
I do use 
"file_directory_format": "{model_username}",
I just need the content creators name option - would be great.
so would be
"file_directory_format": "{name} @{model_username}",

https://onlyfans.c0m/kianna_dior would be:
Kianna Dior @kianna_dior

would really save some time atleast for me as I do rename all folders to that format after scraping.
